Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -247,6 +252,21 @@ public override void Enable()
             InputEnabled?.Invoke();
         }
 
+        public override void LateUpdate()
+        {
+            // Check whether manual initialization of input module is needed.
+            // The check is only required once after input system is created.
+            if (!isInputModuleAdded && !inputModuleChecked)
+            {
+                if (inputModule.ManualInitializationRequired)
```

Comment:
should we verify that inputModule is not null here?

Reply:
My understanding is that inputModule cannot be null there unless there is something wrong with other code that leads to that scenario, so I left out the null check.